client/views/changelog.tpl

```diff
@@ -0,0 +1,27 @@
+{{#if latest}}
+	<h3>New version available!</h3>
+	<p>
+		There is a new {{#if latest.prerelease}}pre-{{/if}}release <b>{{latest.version}}</b> of The Lounge available.
+		<a href="{{latest.url}}" target="_blank" rel="noopener">View details on GitHub</a>
+	</p>
+	<p>We strongly suggest updating The Lounge to the latest version.</p>
+	<hr>
+{{/if}}
+
+{{#if gitCommit}}
+	<p>The Lounge is running from source (commit <a href="https://github.com/thelounge/lounge/tree/{{gitCommit}}" target="_blank" rel="noopener"><code>{{gitCommit}}</code></a>) based on <strong>{{current.version}}</strong></p>
+	<p><a href="https://github.com/thelounge/lounge/compare/{{gitCommit}}...master" target="_blank" rel="noopener">View changes from <code>{{gitCommit}}</code> to <code>master</code> on GitHub</a> to see the changes you are missing</p>
```

Comment:
"Compare changes between <gitCommit>, <version>, and <master> to see what you're missing"